packages/report-server/src/reportBuilder/output/functions/rawDataExport/rawDataExportBuilder.ts

```diff
@@ -50,7 +50,7 @@ export class RawDataExportBuilder {
           const { dataElements } = await this.aggregator.fetchDataGroup(surveyCode);
           return dataElements.map(dataElement => ({
             key: dataElement.code,
-            title: dataElement.text,
+            title: dataElement.text || dataElement.name || '',
           }));
```

Comment:
This part of the code will be removed anyway in `data-tables-ph1`, so I think it is fine to go with the quickest way.